data.sql

```diff
@@ -15,4 +15,20 @@ VALUES
 ('Angemon', '2005-06-12', 1, true, -45),
 ('Boarmon', '2005-06-07', 7, true, 24.4),
 ('Blossom', '1998-10-13', 3, true, 17),
-('Ditto', '2022-05-14', 4, true, 22);
\ No newline at end of file
+('Ditto', '2022-05-14', 4, true, 22);
+
+--INSERT DATA INTO owners TABLE
+INSERT INTO owners (full_name, age)
+VALUES
+('Sam Smith', 34),
+('Jennifer Orwell', 19),
+('Bob', 45),
+('Melody Pond', 77),
+('Dean Winchester', 14),
+('Jodie Whittaker', 38);
+
+--INSERT DATA INTO species TABLE
+INSERT INTO species (name)
```

Comment:
- [ ] You missed a very key requirement that should be done on the `owner`. Kindl find them below, and please add the screenshot to the pull request

### Missed requirement:
![image](https://github.com/BenMKT/vet_clinic_database/assets/50990925/8f49e5a6-75a8-4088-8897-a4b1f0da30bc)

Reply:
- Moved files from `queries.sql` file to `data.sql` file to resolve above.

Reply:
This change has been done well